tests/utils.go

```diff
@@ -2235,6 +2235,15 @@ func NewRandomFedoraVMIWitGuestAgent() *v1.VirtualMachineInstance {
 	)
 }
 
+func NewRandomFedoraVMIWitGuestAgentNetworkData(networkData string) *v1.VirtualMachineInstance {
```

Comment:
I would squash the last three commits which all are doing the same-  changing the fedora vmis to use sateful dhcp.

Reply:
Done.